flw/src/main/java/org/motechproject/nms/flw/service/FrontLineWorkerService.java

```diff
@@ -0,0 +1,21 @@
+package org.motechproject.nms.flw.service;
+
+import org.motechproject.nms.flw.domain.FrontLineWorker;
+
+import java.util.List;
+
+/**
+ * Simple example of a service interface.
```

Comment:
We should clean up these placeholder comments from the archetype. Ok to punt until #53 if you want though.